ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -30,6 +30,7 @@
 - [Spanning Tree](#spanning-tree)  
 - [Internal VLAN Allocation Policy](#internal-vlan-allocation-policy)  
 - [VLANs](#vlans)  
+- [Virtual Router MAC Address](#virtual-router-mac-address)  
```

Comment:
Since this stanza is after interfaces in configuration, we should apply the same schema in documentation:

```
ip virtual-router mac-address 00:1c:73:00:dc:01
!
ip address virtual source-nat vrf TENANT_A_PROJECT02 address 10.1.255.3
!
ip routing
```

--> First section in `Routing` block.